go/test/endtoend/vtgate/queries/subquery/subquery_test.go

```diff
@@ -170,7 +170,7 @@ func TestSubqueryInAggregation(t *testing.T) {
 	mcmp.Exec("insert into t1(id1, id2) values(0,0),(1,1)")
 	mcmp.Exec("insert into t2(id3, id4) values(1,2),(5,7)")
 	mcmp.Exec(`SELECT max((select min(id2) from t1)) FROM t2`)
-	mcmp.Exec(`SELECT max((select group_concat(id1, id2) from t1 where id1 = 1)) FROM t1 where id1 = 1`)
+	mcmp.Exec(`SELECT max((select group_concat(id1, id2) from t1 where id1 = 1)) as x FROM t1 where id1 = 1`)
```

Comment:
why an alias is needed here?

Reply:
Without the alias we will not normalize the subquery, since it is in a SELECT expression. We are however normalizing the `WHERE` clause, so the planner doesn't know that the outer query and the inner query are going to the same shard. That turns this into a correlated query that we don't support. By adding the alias, we are free to normalize both and the planner knows it safe to merge the two sides of the subquery

Reply:
can you add a failing query to the `unsupported` list if it is not already there.